modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -187,12 +189,12 @@ const SelectButton = styled('button')<Pick<SelectBaseProps, 'error' | 'grow' | '
       },
     },
   },
-  ({error, isMenuHidden}) => {
+  ({error, isMenuHidden, isMenuHiding}) => {
```

Comment:
This isn't relevant to this PR, but I really don't like that `isMenuHidden` and `isMenuHiding` are decoupled bits of state. This feels like something we should fix later on. I would opt for a `menuVisibility` state that can either be `visible`, `hiding`, or `hidden`.

Reply:
Good call, might be a quick fix I can address in this PR -- otherwise, I'll make a note of it for a later PR (or maybe package it into #641 since I'm already making adjustments to the menu there).

Reply:
Yeah, no worries. I don’t want to introduce scope creep on what would be an isolated change. Totally up to you. 👍